gemd/units/impl.py

```diff
@@ -36,8 +39,13 @@ def _scaling_preprocessor(input_string: str) -> str:
         if not exponent and token.type == NUMBER:
             scales.append([token.string, False])
             tight_division = division
-        exponent = token.type == OP and token.string in {"^", "**"}
-        division = token.type == OP and token.string in {"/", "//"}
+        if token.type == OP:
+            if token.string not in {"+", "-", "*", "/", "//", "^", "**", "(", ")"}:
```

Comment:
This would be more readable if the set of all allowed tokens was broken out as a constant

Reply:
Fair.  I'd wanted to import it from Pint, but it's a private variable.